README.md

```diff
@@ -9,7 +9,19 @@ but also make certain assumptions that you may not want to impose on your module
 
 ## Release Notes
 
-### veresion 1.7.0
+### version ???
```

Comment:
Should we change this version number?

Reply:
This seems a reasonably clear version number to me. :-) (Yes, thanks for the reminder.)